src/components/accessories/splitButton/styles.scss

```diff
@@ -0,0 +1,27 @@
+@import "../../../styles/variables";
+
+// .buttonGroup {
+
+// .span {
+//   background-color: $c-color-primary !important;
+//   color: $c-white !important;
+//   height: 30px !important;
+//   width: 60px !important;
+// }
+
+// .ohsplitbutton {
+//   // background-color: $c-color-primary !important;
+//   // color: $c-white !important;
+//   // border-radius: 30px !important;
+//   // height: 30px !important;
+//   // width: 60px !important;
+//   // margin: 10px !important;
+// }
+
+// .button__label {
+//   // font-size: 9px !important;
+//   // font-weight: bold;
+// }
+// .ArrowDropDownIcon {
+//   background-color: $c-color-primary;
+// }
```

Comment:
There is no reason to maintain this file, remove it. 

In general: **remove each single commented code in each file** unless it gonna be use later (in that case use put a TODO inside the comment).